### DISCUSSION TITLE
Usando una función para kata 4 ejercicio 2

### DISCUSSION THREAD

**Discussion by contjoseph** [LaunchX-InnovaccionVirtual/CursoIntroPython#105] [Show and tell]
Buenas tardes a tod@s. Para la Kata 4 ejercicio 2 yo utilize una funcion y arroja el mismo resultado. 
Es valido resolver el ejercicio de esta manera? Gracias de antemano. 

def datos_sobre_gravedad(moon, planet, gravedad):
    print("Gravity Facts about " + moon)
    print("-" * 80)
    print("Planet Name: " + planet)
    print("Gravity on " + moon + ": " + str(gravedad * 1000) + " m/s2")
datos_sobre_gravedad("Moon", "Earth", 0.00162)

**Comment by OscarPedroza** [comment]
Hola buenas noches, la sherpa fer comento que no hay problema en la forma en la que se resuelva, simplemente se dan las bases y las maneras de resolver son validas siempre y cuando den con el resultado requerido.

**Comment by ascamarena** [comment]
Las instrucciones dicen que primero resuelvas (o lo intentes) y luego mires la solución. Supongo que si lo haces de otra manera está bien, no creo que importe el camino si se llega al mismo lugar 🏝🛥.

Aunque en este caso concreto, creo que la kata solicita utilizar el .format() y la generación de la plantilla. No tanto a resolver la gravedad.

PD: El código del compañero cumple con ambos ⭐
